src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/15.diff.sql

```diff
@@ -0,0 +1,98 @@
+﻿/*************************************************************
+    Background job checkpoint feature
+**************************************************************/
+
+/*************************************************************
+    Background job checkpoint table
+**************************************************************/
+
+IF NOT EXISTS (SELECT 1 FROM sys.tables WHERE name = 'BackgroundJobCheckpoint')
+BEGIN
+     CREATE TABLE dbo.BackgroundJobCheckpoint
```

Comment:
>Background [](http://example.com/codeflow?start=22&length=10)

I prefer to have more meaningful name. `Background` sound like implementation details which  can be omited.

Reply:
Updated to EventAgentCheckpoint since that will be the application writing to this table. I was initially trying to keep the name generic enough so that other background or long running jobs could use it, but am also okay with naming it something more specific.